modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2129,6 +2129,33 @@ TEST_P(Test_ONNX_nets, Alexnet)
     expectNoFallbacksFromIE(net);
 }
 
+// Waiting for https://github.com/opencv/opencv/pull/24386 for a bug fix to get correct result
+TEST_P(Test_ONNX_nets, RAFT)
+{
+    std::string weight_path = _tf("models/optical_flow_estimation_raft_2023aug.onnx", false);
+    std::string img0_path = findDataFile(std::string("gpu/opticalflow/frame0.png"));
+    std::string img1_path = findDataFile(std::string("gpu/opticalflow/frame1.png"));
+
+    Size target_size{480, 360};
+    auto img0 = imread(img0_path);
+    auto img1 = imread(img1_path);
+    auto blob0 = blobFromImage(img0, 1.0, target_size, 0, true);
+    auto blob1 = blobFromImage(img1, 1.0, target_size, 0, true);
+
+    auto net = readNet(weight_path);
+    net.setInput(blob0, "0");
+    net.setInput(blob1, "1");
+    std::vector<std::string> outnames{"12007", "12006"};
+    std::vector<Mat> outs;
+    net.forward(outs, outnames);
+
+    // output 12006 is not checked to save space in opencv_extra since its ref is > 1MB,
+    // and output 12006 is calculated from 12007 so checking 12007 is sufficient.
+    std::string ref_12700_path = _tf("data/output_optical_flow_estimation_raft_2023aug.npy");
+    auto ref0 = blobFromNPY(ref_12700_path);
+    normAssert(ref0, outs[0], "", 1e-5, 1.4e-4);
```

Comment:
https://github.com/opencv/opencv/actions/runs/6506670391/job/17672657622?pr=24092
```
[ RUN      ] Test_ONNX_nets.RAFT/0, where GetParam() = OCV/CPU
/Users/opencv-cn/GHA-OCV-1/_work/opencv/opencv/opencv/modules/dnn/test/test_common.impl.hpp:79: Failure
Expected: (normInf) <= (lInf), actual: 0.000179291 vs 0.00014
  |ref| = 0.47372817993164062
[  FAILED  ] Test_ONNX_nets.RAFT/0, where GetParam() = OCV/CPU (1957 ms)
[ RUN      ] Test_ONNX_nets.RAFT/1, where GetParam() = OCV/CPU_FP16
/Users/opencv-cn/GHA-OCV-1/_work/opencv/opencv/opencv/modules/dnn/test/test_common.impl.hpp:79: Failure
Expected: (normInf) <= (lInf), actual: 0.000179291 vs 0.00014
```

Reply:
Thank you for the note. Already tuned the threshold.